Basic/Invaders/Assets/Scripts/EnemyBullet.cs

```diff
@@ -82,8 +90,8 @@ void SpawnExplosionVFXClientRpc(Vector3 spawnPosition, Quaternion spawnRotation)
     private void OnGameOver(bool oldValue, bool newValue)
     {
         enabled = false;
-
+        
```

Comment:
```suggestion

```
accidental trailing whitespace

Reply:
thinking about this, it might be worth to add a Standards check job to our CI like we have in BR. Not relevant for this specific PR though, but I'd recommend creating a ticket on Jira to keep track of it

Reply:
Created it here: [MTT-5663](https://jira.unity3d.com/browse/MTT-5663).